commands/static_syncer.go

```diff
@@ -56,6 +57,9 @@ func (s *staticSyncer) syncsStaticEvents(staticEvents []fsnotify.Event) error {
 		syncer.ChmodFilter = chmodFilter
 		syncer.SrcFs = sourceFs.Fs
 		syncer.DestFs = c.Fs.Destination
+		if c.renderStaticFilesToDisk {
+			syncer.DestFs = afero.NewOsFs()
```

Comment:
I don't think this is good enough. There's more to `c.Fs.Destination` than just the filesystem; in server mode we also (sometimes) wraps it with a change detector and a duplicate counter. So I suggest you add a `Fs.DestinationStatic` that mostly gets to be identical to the regular one.

Reply:
fixed